src/creature_states.c

```diff
@@ -175,8 +175,8 @@ struct StateInfo states[CREATURE_STATES_COUNT] = {
     0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,  CrStTyp_Work, 0, 0, 0, 0,  0, 0, 0, 1},
   {imp_digs_mines, NULL, NULL, NULL,
     0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,  CrStTyp_Work, 0, 0, 0, 0,  0, 0, 0, 1},
-  {NULL, NULL, NULL, NULL,
-    0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,  CrStTyp_Move, 0, 0, 1, 0,  0, 0, 0, 1},
+  {creature_casting_preparation, NULL, NULL, NULL,
+    1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,  CrStTyp_Work, 0, 1, 0, 0,  0, 0, 0, 1},
```

Comment:
Did you check all of these to be correct? Or from which state did you copy these from?

Reply:
I refer to CrSt_CreatureSlapCowers (creature_slap_cowers()) because my new state is similar. They are both very short temporary states and the creature needs to resume the original states soon.
I didn't test every member. I just examined the definition of StateInfo and had a basic understanding of them.